test/Dapr.E2E.Test/E2ETests.cs

```diff
@@ -83,22 +83,7 @@ public Task DisposeAsync()
 
         protected async Task WaitForActorRuntimeAsync(IPingActor proxy, CancellationToken cancellationToken)
         {
-            while (true)
-            {
-                this.Output.WriteLine($"Waiting for actor to be ready in: {this.AppId}");
-                cancellationToken.ThrowIfCancellationRequested();
-
-                try
-                {
-                    await proxy.Ping();
-                    this.Output.WriteLine($"Found actor in: {this.AppId}");
-                    break;
-                }
-                catch (DaprApiException)
-                {
-                    await Task.Delay(TimeSpan.FromMilliseconds(250));
-                }
-            }
+            await ActorRuntimeChecker.WaitForActorRuntimeAsync(this.AppId, this.Output, proxy, cancellationToken);
```

Comment:
Suggestion: would it make sense to push this down to `DaprTestAppLifecycle`?

Reply:
Not yet, most of the apps (3/4) are still using the E2ETests as a base and don't interact with the `DaprTestAppLifecycle`. Once we move more apps to it, it'd make more sense.